src/CSS/InfoPages.css

```diff
@@ -0,0 +1,26 @@
+.fixed-logo {
+    position: fixed; 
+    top: 80px;
+    left: 50%;
+    transform: translateX(-50%);
+    z-index: 1000;
+}
+
+.white-banner {
+    position: fixed;
+    top: 0;         /* start from the very top of the screen */
```

Comment:
The comments here are a bit thorough ^^ Probably fine to just delete all the `/* ... */`